.travis.yml

```diff
@@ -55,8 +55,5 @@ script:
    - travis_wait 30 ./gradlew :libs:SalesforceHybrid:connectedAndroidTest
    - ./gradlew :libs:SalesforceReact:assembleDebug
    - travis_wait 30 ./gradlew :native:NativeSampleApps:RestExplorer:connectedAndroidTest
-   # make sure the screen is unlocked
-   - adb shell input keyevent 82
-   - adb shell input keyevent 84
-   - adb shell input keyevent 84
-   - travis_wait 30 ./gradlew :native:TemplateApp:connectedAndroidTest
+   # disable this ui test until fix the unstable test failure 
+   - ./gradlew :native:TemplateApp:assembleDebug
```

Comment:
My motivation to disable the unstable tests is since we don't have a way to summary the test results yet, so it is will be simple to let people check there are failure or not and decide can merge the change or not. Currently, since there are some unstable failure (even it just comes out recently somehow, need more investigation), people still do merge does matter the result is green or not. We want to make the PR build result as a gate for PR merge. And so far there are couple of regressions, even just in test, need to update/modify test codes or something on travis ci. The author of the checkin is the best/effective person to fix the new failure.